C-Plus-Plus/sort/DNF_sort.cpp

```diff
@@ -0,0 +1,70 @@
+// Created by shubhgupta9
+//C++ implementation of DNF sort or Dutch National Flag sort
+
+//code for the algorithm goes here:-
+
+/* This algorithm is used to sort an array of 3 numbers, in our example code we will be taking 
+a default unsorted array which consists of 0s,1s and 2s.
+*/
```

Comment:
- Add Multi-Line Comments

Reply:
Thank you sir for reviewing my pull request.
I have done all the required changes.
I am not able to edit the README file please help me regarding that.
Thank You

Reply:
Done Sir. You can see all the required changes.
Thank You for the review.